code/_onclick/hud/constructs.dm

```diff
@@ -0,0 +1,32 @@
+/mob/living/simple_animal/hostile/construct/create_mob_hud()
+    if(client && !hud_used)
+        hud_used = new /datum/hud/construct(src)
+
+/datum/hud/construct/New(mob/owner)
+	..()
+
+	var/constructtype
+
+	if(istype(mymob,/mob/living/simple_animal/hostile/construct/armoured) || istype(mymob,/mob/living/simple_animal/hostile/construct/behemoth))
```

Comment:
i'm not sure which is worse
an istype chain for subtypes
or adding 4 new huds for no reason